README.md

```diff
@@ -7,7 +7,7 @@
 ChaosBot is a social coding experiment to see what happens when the absolute
 direction of a software project is turned over to the open source community.
 
-![There was clearly a kitty missing.](http://thecatapi.com/api/images/get?format=src&type=png&size=small)
+![Image of a kitten with overlay: "I WONDER IF BEING SANE MEANS DISREGARDING THE CHAOS THAT IS LIFE...PRETENDING ONLY AN INFINITESIMAL SEGMENT OF IT IS REALITY" Source: http://existentialpets.tumblr.com/post/48069725344](assets/chaos-kitty.jpg)
```

Comment:
>and changes cause vote count to reset.
can you point to the lines that are causing that? I cannot find them and I would like it to be more intelligent than just reset